README.md

```diff
@@ -36,6 +36,7 @@ This repository is an index for several projects providing *great prepackaged/pr
   - RPM
     - `dnf` | `yum` ([Fedora](https://getfedora.org))
       - [src.fedoraproject.org](https://src.fedoraproject.org/)
+      - [copr.fedorainfracloud.org](https://copr.fedorainfracloud.org/coprs/rezso/HDL) EPEL & Fedora weekly latest edge builds
```

Comment:
As commented in hdl/containers#41, there are multiple repos/projects in copr which provide EDA tools. Therefore, it doesn't feel correct pointing `copr.fedorainfracloud.org` to one of them only. Please, create a list (with a single item for now):

```
- [copr.fedorainfracloud.org](https://copr.fedorainfracloud.org)
  - [rezso/HDL](https://copr.fedorainfracloud.org/coprs/rezso/HDL) (CentOS and Fedora; x86_64, aarch64 and ppc64le)
```

- [copr.fedorainfracloud.org](https://copr.fedorainfracloud.org)
  - [rezso/HDL](https://copr.fedorainfracloud.org/coprs/rezso/HDL) (CentOS and Fedora; x86_64, aarch64 and ppc64le)
  
---

By the way, you might also want to add a section in `CONTEXT.md`, to provide a brief description of why that copr project exists. What use cases is it covering which are not supported by other packaging solutions? Or which are the advantages of using it even if other packaging solutions exist?

I guess the obvious response is that it's a "native" repository for CentOS and Fedora users, which provides bleeding-edge versions of the tools, compared to the less frequently updated upstream packages.

Reply:
> As commented in [hdl/containers#41](https://github.com/hdl/containers/issues/41), there are multiple repos/projects in copr which provide EDA tools. Therefore, it doesn't feel correct pointing `copr.fedorainfracloud.org` to one of them only. Please, create a list (with a single item for now):

- Right. Corrected the URL hinting as requested, now it should be **one-of-them** (leaves space other possible copr entries).

> By the way, you might also want to add a section in `CONTEXT.md`, to provide a brief description of why that copr project exists.

- CONTEXT.txt looks to me as a generalistic intro into the world of packaging systems.

> What use cases is it covering which are not supported by other packaging solutions? Or which are the advantages of using it even if other packaging solutions exist?

- DNF/YUM (tools) as RPM (native format) is a specific thing to RedHat/Fedora (briefly explained in CONTEXT.md) see no point of even comparing it.
 
> I guess the obvious response is that it's a "native" repository for CentOS and Fedora users, which provides bleeding-edge versions of the tools, compared to the less frequently updated upstream packages.

- Summary brief description of the repository purpose, cycles & meaning are already given at the [rezso/HDL mainpage](https://copr.fedorainfracloud.org/coprs/rezso/HDL)

I cannot say it's best or *only* or other kind of promotions, would like let user decide it.
I am a simple consumer happy with a "corner" that help me to fit these pieces together for my end-needs (possibly others).